chart/templates/proxy_injector.yaml

```diff
@@ -81,6 +81,9 @@ rules:
 - apiGroups: [""]
   resources: ["namespaces"]
   verbs: ["get"]
+- apiGroups: ["", "extensions"]
+  resources: ["replicationcontrollers", "replicasets"]
+  verbs: ["get"]
```

Comment:
If you take my `GetOwnerKindAndName` suggestion below, then the proxy injector won't need get privileges for replicationcontrollers, but it will need list and watch privileges for replicasets. I'd change this to:

```
- apiGroups: ["extensions"]
  resources: ["replicasets"]
  verbs: ["list", "get", "watch"]
```